subt/ros/robot/launch/freyja.launch

```diff
@@ -187,11 +188,20 @@
                                    true,  true,  true,
                                    true,  true,  true] </rosparam>
 
+    <!-- The unit of the threshold is a Mahalanobis distance (i.e. sum) of numbers of sigmas in each dimension -->
+    <rosparam param="odom0_twist_rejection_threshold">4</rosparam>
+    <rosparam param="odom2_twist_rejection_threshold">4</rosparam>
+    <rosparam param="odom3_twist_rejection_threshold">4</rosparam>
+    <rosparam param="odom4_twist_rejection_threshold">4</rosparam>
```

Comment:
Let's focus on fixing slipping wheels. Let's set the rejection threshold for odom0 and for nothing else.

Reply:
OK, let's give it one more try/day ...